layouts/index.hbs

```diff
@@ -52,7 +52,7 @@
 
             <ul class="list-divider-pipe home-secondary-links">
               <li>
-                <a href="https://nodejs.org/{{site.locale}}/download/current/">{{ labels.other-downloads }}</a>
+                <a href="download/current/">{{ labels.other-downloads }}</a>
```

Comment:
nit: since the others are site absolute
```suggestion
                <a href="/download/current/">{{ labels.other-downloads }}</a>
```

Reply:
Needs to include the language `{{site.locale}}` otherwise the path doesn't resolve (i.e. https://nodejs.org/download/current doesn't work). Have pushed that up to this commit too now.